source/CMakeLists.txt

```diff
@@ -144,3 +145,14 @@ endif()
 if(TARGET PoseConversions)
     add_dependencies("PoseConversions" CopyHandEyeFiles)
 endif()
+
+add_custom_target(
+    CopyCameraSettings
+    COMMAND ${CMAKE_COMMAND} -E copy_directory
+            ${CMAKE_CURRENT_SOURCE_DIR}/Settings/
+            ${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_CFG_INTDIR}/Settings
+)
+if(TARGET CaptureHDRLoop)
+    message(STATUS "Copy resources for : ${TARGET}")
```

Comment:
Leftover debug print? Don't think we need this, we're not using it for the other things we copy.